app/javascript/packs/puerto-rico-guidance.ts

```diff
@@ -13,5 +13,56 @@ function onStateSelectionChange() {
   showOrHidePuertoRicoExtras(stateSelector?.value);
 }
 
+function onIdentityDocStateSelectionChange() {
```

Comment:
Could we generalize this a bit more so that maybe we add a class name to all of the inputs where we want this behavior, rather than looking for IDs for each of them?

Thinking something like...

```ts
const stateSelectors = document.querySelectorAll<HTMLSelectElement>('.state-selector');
stateSelectors.forEach((stateSelector) => {
  stateSelector.addEventListener('change', () => showOrHidePuertoRicoExtras(stateSelector.value));
  showOrHidePuertoRicoExtras(stateSelector.value);
});
```

Reply:
I support this suggestion!